app/src/main/java/me/saket/dank/ui/user/UserSessionRepository.java

```diff
@@ -22,20 +32,26 @@
 public class UserSessionRepository {
```

Comment:
I am not sure if `UserSessionRepository` is the best place to store this information. This class is meant to only manage the logged in user's information and nothing else. 

I wonder if we should store it in `SubredditActivity` directly.